### DISCUSSION TITLE
What is STLForecaster model?

### DISCUSSION THREAD

**Discussion by ckwan48** [alan-turing-institute/sktime#3180] [Q&A]
I have been using STLForecaster to forecast but it doesn't specify what kind of model you would use to forecast each component after it has been separated. It states that it is based off of statsmodels STLForecast and you can specify the model. Does anyone know what kind of model sktime is using?

**Comment by fkiraly** [comment]
I've updated the docstrings here: https://github.com/alan-turing-institute/sktime/pull/3203

Apologies, the docstrings were a bit unclear.

If you look into the PR, I hope it's clear what the algorithm is now.

The default models are `NaiveForecaster` instances with different parameters, as the docstring explains (the defaults were already explained in the old docstring)
